twilight-validate/src/component.rs

```diff
@@ -1111,6 +1200,7 @@ mod tests {
             channel_types: None,
             custom_id: "custom id 2".into(),
             disabled: false,
+            default_values: None,
```

Comment:
should we test default values validation? the function seems straightforward enough for it to not require a test but im not sure

Reply:
I've added unit tests with the validation code moved to two separate functions. Please let me know whether that's what you were thinking of :slightly_smiling_face:

Reply:
yes it was, thank you :)